.gitignore

```diff
@@ -0,0 +1,3 @@
+# cache files
+__pycache__/
+img/__pycache__/
```

Comment:
could it just be `*__pycache__`?

Reply:
right, `git` looks into parent folders for `.gitignore` files, so the first line is sufficient. Have changed it

Reply:
Oh, and the asterix would be superfluous, I think, because it doesn't expand to folders (AFAIK), and the cache files have a specified name

Reply:
yes, no asterix (sic ;) )  required. thanks!